### PR TITLE
322 compare resilience metrics

### LINKED ISSUE #322: Compare Resilience Metrics
Just reuse all the comparison tables from various steps

### PR DESCRIPTION
**Pull request by longshuicy**
- Add resilience metrics and set to default
- Resilience metrics compare is using various compare tables
- Clean up the diff and percentage diff
- Flip order bug: Diff = B - A

### CODE REVIEW

File: src/components/childComponents/outputWidgets/PopDislocationCompare.jsx
Comment (longshuicy) [inline src/components/childComponents/outputWidgets/PopDislocationCompare.jsx:135]: @ylyangtw I hope you don't mind. I use the colSpan instead of "empty" cell here

Comment (ylyangtw) [reply src/components/childComponents/outputWidgets/PopDislocationCompare.jsx:135]: Nice!

**Comment by ylyangtw** [comment]
Looks good. I have a question, do we want to keep the image download? The downloaded image looks a bit odd. Maybe we dont need it?
image

**Review by ylyangtw — APPROVED** [review]
Nice cleaning! Thanks much!

**Comment by longshuicy** [comment]
> Looks good. I have a question, do we want to keep the image download? The downloaded image looks a bit odd. Maybe we dont need it? image

I actually don't know.. i was hoping maybe the designer can help us with that (if we needed, and if we do where to place it)

**Comment by ylyangtw** [comment]
> > Looks good. I have a question, do we want to keep the image download? The downloaded image looks a bit odd. Maybe we dont need it? image
> 
> I actually don't know.. i was hoping maybe the designer can help us with that (if we needed, and if we do where to place it)

I see. Then let's wait for final review:)

**Pull request merged**
